__tests__/app.test.js

```diff
@@ -135,3 +135,60 @@ describe('/api/articles/:article_id', () => {
       });
   });
 });
+
+describe('/api/articles/:article_id/comments', () => {
+  test('GET: 200 sends an array of comments for the given article_id', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+        expect(body.comments.length).toBe(11);
+      });
+  });
+
+  test('GET: 200 sent comments have the required proerties', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.comments;
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+  test('GET: 200 sends an array of sorted comments by date DESC by default', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy('created_at', {
+          descending: true,
+        });
+      });
+  });
+  test('GET: 200 sends an empty array if the article exists but there are no associated comments', () => {
+    return request(app)
+      .get('/api/articles/2/comments')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toEqual([]);
+      });
+  });
+  test('GET:404 sends an appropriate status and error message when the article does not exist', () => {
+    return request(app)
+      .get('/api/articles/99999/comments')
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe('article does not exist');
+      });
+  });
```

Comment:
Killing it so far in the tests Aisling, just missing an test to check if you get a 400 if given an article with an invalid ID
like `/api/articles/katherine/comments`